plugins/secrets/validators/SecretsValidators.ts

```diff
@@ -16,32 +16,18 @@ interface ServiceError {
 
 const secretPathInvalidCharacters = /[\0<>|:&]/;
 const secretPathParentDirReference = /(\.\.)/;
-
-function isRelativeFilePath(path: string) {
-  return !(
-    path[0] === "/" ||
-    path[path.length - 1] === "/" ||
-    path[0] === "\\" ||
-    path[path.length - 1] === "\\"
-  );
-}
+const secretPathDirectoryPath = /\/$/;
```

Comment:
not sure whether that's a good thing to check. 
i'm mostly in favour of letting the backend check stuff. but let's try.